admin-dev/themes/new-theme/.webpack/common.js

```diff
@@ -109,6 +109,7 @@ module.exports = {
     alias: {
       vue$: 'vue/dist/vue.common.js',
       app: path.resolve(__dirname, '../js/app'),
+      psvue: path.resolve(__dirname, '../js/vue'),
```

Comment:
Just notice that `app` is already for vue components / apps..
You should move everything in this `app` directory

Reply:
we should clean up these directories.
Having `js/app` storing all our BO apps, and `js/vue` storing all our vue apps / globals `components`, shared `store`, etc wdyt?